### ISSUE TITLE
Autolayout Constraints Issue

### ISSUE THREAD

**Issue by QuynhNguyen** [ninjaprox/NVActivityIndicatorView#117]
```
[LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
	(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSAutoresizingMaskLayoutConstraint:0x61800009e140 h=--& v=--& UILabel:0x7fd27a938df0'Signing you in!'.midY == 486.912   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x61800009dba0 h=--- v=--- UIWindow:0x7fd277504620.height == 736   (active)>",
    "<NSLayoutConstraint:0x61800009e280 UILabel:0x7fd27a938df0'Signing you in!'.centerY == UIView:0x7fd27a93a8f0.centerY   (active)>",
    "<NSLayoutConstraint:0x61800009cb10 V:|-(0)-[UIView:0x7fd27a93a8f0]   (active, names: '|':UIWindow:0x7fd277504620 )>",
    "<NSLayoutConstraint:0x61800009b490 UIView:0x7fd27a93a8f0.bottom == UIWindow:0x7fd277504620.bottom   (active)>"
)

```

**Comment by ninjaprox** [comment]
@QuynhNguyen: What version are you using? If the latest, I guess it might be related to this #116.

**Comment by QuynhNguyen** [comment]
Yup, latest version

**Comment by ninjaprox** [comment]
@QuynhNguyen: Thanks for reporting. Fixed in `3.4`.
